### DISCUSSION TITLE
The following error originated from your test code, not from Cypress.  > process is not defined error

### DISCUSSION THREAD

**Discussion by Mary1210** [cypress-io/cypress#29283] [General]
Hello,
I added url in environment variables in cypress.config.js
as per this photo
![image](https://github.com/cypress-io/cypress/assets/45977095/c4fdf6ff-d1b9-438d-b94f-00d9204b6946)

then I use the environmet variable in my test case as per the following photo
![image](https://github.com/cypress-io/cypress/assets/45977095/61175a90-82c6-45fa-86db-928b39deabfc)

When I run my test case, I got this error as per the following photo
![image](https://github.com/cypress-io/cypress/assets/45977095/8b24d71e-fdc1-47e1-9447-04f2fed1aa82)

Is there any solution for this?
Note: I used cpress 13.7.2

**Comment by Clement-Bresson** [comment]
Same error here
